lk2nd/device/dts/msm8952/msm8937-motorola-jeter.dts

```diff
@@ -6,20 +6,42 @@
 / {
 	qcom,msm-id = <QCOM_ID_MSM8937 0x00>;
 	qcom,board-id = <0x4A 0x8100>,
-			<0x4A 0x8000>;
+			<0x4A 0x8000>,
+			<0x4B 0x8000>;
 
 	/* 
 	 * model is required by bootloader to pick dtb.
 	 * The bootloader also crashes if model isn't present in every dtb. (see lk2nd.dtsi for more info)
+	 * Build with: make TOOLCHAIN_PREFIX=arm-none-eabi- SIGN_BOOTIMG=1 LK2ND_ADTBS="msm8937-motorola-jeter.dtb" LK2ND_DTBS="" lk2nd-msm8952
 	 */
 	
 	model = "jeter";
 };
 
 &lk2nd {
-	model = "Motorola Moto G6 Play (jeter)";
+	model = "Motorola Moto G6 Play (jeter) (MSM8937)";
 	compatible = "motorola,jeter";
 	lk2nd,match-device = "jeter";
 
 	lk2nd,dtb-files = "msm8937-motorola-jeter";
+
+	gpio-keys {
+		compatible = "gpio-keys";
```

Comment:
the whole node looks like defaults for 8937 platform. Are you sure you need this? I prefer to omit keycode overrides if they do nothing.

Reply:
Not sure, I am waiting for @pryzhak for feedback to remove gpio-keys from both DTS to see if their keys still work, I added them before the device even booted up.